joomla/joomla-cms
# Update composer libraries

Pull Request for Issue #10835  .
#### Summary of Changes

Updates phpmailer, joomla registry, symfony polyfill (php 5.6), symfony yaml to their latest versions
#### Testing Instructions

Registry is used everywhere. So navigate around and check for errors. Yaml isn't used by core. If possible test in PHP < 5.6 to test the polyfill and check emails still send for phpmailer.
